presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcFileRewriter.java

```diff
@@ -71,6 +73,12 @@
 
     private File temporary;
 
+    @DataProvider(name = "useOptimizedOrcWriter")
```

Comment:
The new ORC file writer (i.e. OrcFileWriter.java) never get used in `OrcFileRewriter`. In OrcFileRewriter.java:L98, we always create an old ORC writer when rewriting shards. The test here is only testing the case when ORC rewriter use the old ORC file writer to rewrite files written by the new ORC writer.

Reply:
This will be used for `createFileWriter` for now. Later it will be used for rewriter as well.